### ISSUE TITLE
DESIGN_KW and GEN_KW working together

### ISSUE THREAD

**Issue by berland** [equinor/semeio#59]
There is a syntax conflict in `parameters.txt` when parameters are variably set using design matrices (DESIGN2PARAMS) and GEN_KW.  For the Reek FMU template project, this has led to inclusion of a separate forward model (workaround: PARAMETERS_4DESIGN)/parameters_4designkw.py) that modifies `parameters.txt` by removing all prefixes in front of a colon pr. line 

Forward models used in Reek should all be considered included in semeio (or some repo) as this project is copied manually by users.

I am not very comfortable adding an extra forward model to workaround this incompatibility in a template project and we should seek a better solution. 

A possibility is that DESIGN_KW is changed so when a users wants to replace `<SOMEPARAMETER>`, it will give a hit in lines like `SOMEGROUP:SOMEPARAMETER 100` in `parameters.txt`. 

The PR for the pyscal forward model effectively does this using the function `rm_genkw_prefix()` 
https://github.com/equinor/semeio/pull/53/files#diff-4226388281c1739d36a044379a3fab60R171

Same discussions earlier:
https://www.yammer.com/statoil.com/#/Threads/show?threadId=1222922541
https://github.com/equinor/ert/issues/275
https://github.com/equinor/ert/issues/599

**Comment by berland** [comment]
@jcrivenaes @rnyb

**Comment by berland** [comment]
```
#!/usr/bin/env python
"""
Fix for handling multiple global variable files (ipl and yml) when
using GEN_KW update parameters.txt by appending a repeat of
unc. params in a DESIGN_KW friendly format with updated
parameters.txt file, DESIGN_KW can now be used to fill values in
multiple global variable files

Roger Nybo : Jan 2019

"""

from __future__ import print_function

# -----------------------------------
# paths are relative to ert <runpath>

filename = "parameters.txt"  # input file

# -----------------------------------
# Main
# -----------------------------------

print("---------------------------------------")
print(
    "Read parameters.txt file and append all uncertainty "
    + "parameters to file with GEN_KW friendly format"
)

with open(filename, "r") as myfile:
    all_lines = myfile.readlines()

try:
    infile_append = open(filename, "a")
    print("OK:" + filename + " file was found.\n")
except:
    print("-----\nNot OK: file " + filename + " was not found\n-----")

for line in all_lines:
    if not line.startswith("LOG") and ":" in line:
        x = line.split(":")
        infile_append.write(x[1])

infile_append.close()

print(
    "Finished. Appended unc. params. to " + filename + " --> can be used with DESIGN_KW"
)
```

**Comment by markusdregi** [comment]
> A possibility is that DESIGN_KW is changed so when a users wants to replace , it will give a hit in lines like SOMEGROUP:SOMEPARAMETER 100 in parameters.txt.

One might agree or disagree on the current format of output from ERT, but there is quite a bit of value to sticking to one format and namespacing is for sure not evil ;) I would vote towards the above.

**Comment by berland** [comment]
Solved in #163
